build.gradle

```diff
@@ -100,17 +102,23 @@ gradlePlugin {
     }
 }
 
+def isProdPortal = !System.getProperty('gradle.portal.url')
```

Comment:
is `'agp': '7.0.4',` the most current version?

Reply:
This is the lowest supported version in order to provide backwards compatibility. This is intentional.